example/urls.py

```diff
@@ -16,7 +17,8 @@
     url(r'^send-email/$', views.send_email, name='send-email'),
     url(r'^reporting/', include('django_ses.urls')),
 
-    url(r'^event/', 'django_ses.views.handle_event', name='handle_event'),
+    url(r'^bounce/', 'django_ses.views.handle_bounce', name='handle_bounce'),       # Deprecated, see SESEventWebhookView.
```

Comment:
Does this URL log the warning you have above? Would be good to make sure people who still use this endpoint see warnings.

Reply:
@cobang could you follow up on this question?

Reply:
No, I just wanted to understand when the warning happens. Sounds like it will show when a request comes to the old URL.